src/home_robot/home_robot/utils/bullet.py

```diff
@@ -485,8 +485,6 @@ def compute_ik(
         else:
             random_initialization = True
 
-        verbose = True
```

Comment:
I can put this back if desired, but since verbose is being passed into this function, this seems like a debugging line that was unintentionally left in?

Reply:
yeah no problem, thanks!